pkg/ruler/group.go

```diff
@@ -0,0 +1,29 @@
+package ruler
+
+import (
+	"context"
+	"time"
+
+	"github.com/prometheus/prometheus/rules"
+)
+
+// group is a wrapper around a prometheus rules.Group, with a mutable appendable
```

Comment:
I think it's worth noting that `appendable` points to the same struct as `promGroup.opts.Appendable`.

Reply:
👍